task-4/questions-project/src/App.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState, useEffect } from "react";
+import FinalPage from "./FinalPage";
+import Styled from 'styled-components';
+
+const QuestionHeader = Styled.h1`
+  text-align: center;
+  font-size: 26px;
+  margin: 10px;
+  font-weight: bold;
+  font-family: "News Cycle";
+`;
+
+const MainContainer = Styled.div`
+  display: flex;
+  height: 90vh;
+  justify-content: center;
+`;
+
+const Container = Styled.div`
+  display: flex;
+`;
+
+const Answer = Styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 500px;
+  height: 600px;
+  background-color: ${props => props.color || "#f0efef"};
+    border: 1px solid white;
+`;
+
+const AnswerText = Styled.p`
+  font-size: 20px;
+  font-weight: bold;
+  justify-self: flex-end;
+`;
+
+const QuestionImage = Styled.img`
+  margin: 70px 0 30px 0;
+  object-fit: cover;
+  width: 330px;
+  height: 250px;
+`;
+const QUESTIONS = [
+  {
+    question: "Выберите собаку?",
+    answer: {
+      a: {
+        label: "",
+        image: "images/blackpuppy.jpg",
+      },
+      b: {
+        label: "",
+        image: "images/whitepuppy.jpg",
+      },
+    },
+    correct: "A",
+  },
+  {
+    question: "Чем обработать собаку от клещей?",
+    answer: {
+      a: {
+        label: "А) Таблетки Бравекто ",
+        image: "images/bravecto.jpg",
+      },
+      b: {
+        label: "Б) Капли Барс",
+        image: "images/bars.jpg",
+      },
+    },
+    correct: "A",
+  },
+  {
+    question: "Каким кормом кормить собаку?",
+    answer: {
+      a: {
+        label: "А) Педигри",
+        image: "images/pedigree.jpg",
+      },
+      b: {
+        label: "Б) Грандорф",
+        image: "images/grandorf.jpg",
+      },
+    },
+    correct: "B",
+  },
+
+];
+
+const CORRECT_ANSWER_MESSAGE = "Вы ответили верно!";
+
+function App() {
+  const [questionIndex, setQuestionIndex] = useState(0);
+  const [correctAnswer, setCorrectAnswer] = useState(
```

Comment:
you do not need to store correctAnswer via useState, you always know this is QUESTIONS[questionIndex].correct